src/DynamoCoreWpf/Properties/Resources.resx

```diff
@@ -2399,4 +2399,13 @@ Uninstall the following packages: {0}?</value>
   <data name="ResetCPythonButtonToolTip" xml:space="preserve">
     <value>Resets CPython environment by reloading modules.</value>
   </data>
+  <data name="InstalledPackagesExpanderName" xml:space="preserve">
+    <value>Installed Packages</value>
+  </data>
+  <data name="PackagePathsExpanderName" xml:space="preserve">
+    <value>Package Paths</value>
```

Comment:
Do you want to reuse the string `Manage Node and Package Path`? Just curious
![image](https://user-images.githubusercontent.com/3942418/121056749-61bee900-c773-11eb-845e-a4b0c8a4cfdb.png)

Reply:
I was following the design.

Reply:
@mjkkirschner I see. Your call between you and @Jingyi-Wen

Reply:
@Jingyi-Wen thoughts?

Reply:
> @Jingyi-Wen thoughts on changing `package paths` to `node and package paths` or `manage node and package paths`?

Seems like `node and package paths` would be more accurate. (No need to add `manage` since we are not using verbs in other expanders or items.) I'll update the Figma

Reply:
👍